docs/configurations.md

```diff
@@ -101,164 +127,102 @@ The following describes the configurations available for each storage.
 | `scalar.db.jdbc.admin.connection_pool.max_idle`           | Maximum number of connections that can remain idle in the connection pool for admin.                                                                                         | `10`                         |
 | `scalar.db.jdbc.admin.connection_pool.max_total`          | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool for admin. Use a negative value for no limit.              | `25`                         |
 
-If you use SQLite3 as a JDBC database, you must set `scalar.db.contact_points` as follows.
+{% capture notice--info %}
+**Note**
+
+If you use SQLite3 as a JDBC database, you must set `scalar.db.contact_points` as follows, replacing `YOUR_DB` with the URL of your SQLite3 database:
 
 ```properties
 scalar.db.contact_points=jdbc:sqlite:<YOUR_DB>.sqlite3?busy_timeout=10000
 ```
 
-Unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html).
-To avoid frequent errors caused internally by [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy), we recommend setting a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+In addition, unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html). To avoid frequent errors caused internally by [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy), we recommend setting a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+</div>
+</div>
 
 ##### Multi-storage support
 
-ScalarDB supports using multiple storage implementations at the same time.
-You can use multiple storages by specifying `multi-storage` for the `scalar.db.storage` property.
+ScalarDB supports using multiple storage implementations simultaneously. You can use multiple storages by specifying `multi-storage` as the value for the `scalar.db.storage` property.
 
-For details about using multiple storages, see [Multi-storage Transactions](multi-storage-transactions.md).
+For details about using multiple storages, see [Multi-Storage Transactions](multi-storage-transactions.md).
 
 ### ScalarDB Server (gRPC)
```

Comment:
Ditto. This should be `ScalarDB Cluster`.
As we chatted, we should refer to the ScalarDB Cluster doc for the configurations.

Reply:
Similar to [my previous comment](https://github.com/scalar-labs/scalardb/pull/1082#discussion_r1337992879), since this PR is for branches `3.8` and earlier, I think this sentence is OK, right?